### PR TITLE
Create adopters list

### PR DESCRIPTION
**Pull request by todaywasawesome**
We'd like to start tracking end-users and adopters of the GitOps Principles. This is a very simple list but can help us start building the community. 

Signed-off-by: todaywasawesome <dan@codefresh.io>

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Who uses GitOps? 
+
+As the GitOps movement grows, we'd like to hear from organizations who are successfully implementing GitOps! Please send a PR with *your* organization name if you are adopting the [GitOps principles](https://opengitops.dev/#principles) in practice. Please include how you are adopting.
```
Comment (chris-short) [inline ADOPTERS.md:3]: I'd change movement to adoption

GitOps isn't a movement in so much that it's a move to a better way to do things.

**Review by chris-short — CHANGES_REQUESTED** [review]
One tweak

File: ADOPTERS.md (hunk 1 above)
Comment (roberthstrand) [reply ADOPTERS.md:3]: I agree with @chris-short, adoption is a better fit in this case.

Comment (scottrigby) [inline ADOPTERS.md:3]: I do like the idea that GitOps can be seen as a kind of movement, but I do agree this prob isn't the place to drop that suggestion 🙂

Here's a GitHub code suggestion you can commit directly if you want. Includes what @chris-short said, with small grammar tweaks to accompany that.

```suggestion
As GitOps adoption grows, we'd like to hear from organizations that are successfully implementing GitOps! Please send a PR with *your* organization name if you are adopting the [GitOps principles](https://opengitops.dev/#principles) in practice. Please include how you are adopting.
```

File: ADOPTERS.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+Currently, the following organizations are officially adopting GitOps (in alphabetical order). 
+
+1. [Example Organization](https://example.com)
```
Comment (scottrigby) [inline ADOPTERS.md:7]: What about including a simple example for adopters to describe how they're adopting GitOps?

```suggestion
1. [Example Organization](https://example.com) – `Example` has adopted GitOps to manage our X and Y products and their required infrastructure. We use A and B tools/techniques, and participate in the GitOps WG meetings. See this a (blog post/talk) about the details of our journey and current setup: <https://example.com/blog/how-we-do-gitops>
```

**Review by scottrigby — CHANGES_REQUESTED** [review]
A few [suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request). 

PS, reminder for everyone: when committing GitHub suggestions from the UI, be sure to paste in your DCO signoff into the comment textarea. Or use a browser plugin to help https://github.com/scottrigby/dco-gh-ui 😉

File: ADOPTERS.md (hunk 1 above)
Comment (todaywasawesome) [reply ADOPTERS.md]: Yeah fair point, we can remove that. I do like GitOps as a movement but we don't have to plug it here.

**Review by scottrigby — APPROVED** [review]
Woot let's gooooo

**Pull request merged**